src/Utils/useTimer.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+
+export const useTimer = () => {
```

Comment:
Can we have doc comments on this hook? What it is trying to do and for what type of use-cases we could use this for.